pymeasure/instruments/nf/li5660.py

```diff
@@ -0,0 +1,1665 @@
+        ":ROUT?", ":ROUT %s",
+        """ A property that control the input connector.
+            A: Single end voltage (terminal A, 1 V max).
+            AB: Differential voltage (terminal A-B, 1 V max).
+            C: Large amplitude voltage (terminal C, 10 V max).
+            I: Current(terminal I, 1 \u03BCAmax when conversion gain is 1 MV/A,
+                                   10 nAmax when conversion gain is 100MV/A).
+            HF: High frequency voltage (terminal HF, 1 V max). """,
+        validator=strict_discrete_set,
+        values=["A", "AB", "C", "I", "HF"],
+        map_values=False
+    )
+
+    route2_terminals = Instrument.control(
+        ":ROUT2?", "ROUT2 %s",
+        """ A property that control the reference signal source.
+            Ref In: Reference input connector.
+            Internal: Internal oscillator.
+            Signal In: Signal input connector. """,
+        validator=strict_discrete_set,
+        values={"Ref In": "RINP",
+                "Internal": "IOSC",
+                "Signal In": "SINP"},
+        map_values=True
+    )
+
+    ######################################
+    # Sensitivity subsystem, 26 Nov 2020 #
+    ######################################
+    current_ac_range_auto = Instrument.control(
+        ":CURR:AC:RANG:AUTO?", ":CURR:AC:RANG:AUTO %d",
+        """ A property that allow user to set the current sensitivity continuous automatic selection function.
+            When current sensitivity is set automatically, dynamic reserve is also set automatically.
+            On: Enables continuous automatic selection of current sensitivity.
+            Off: Disable continuous automatic selection of current sensitivity.""",
+        validator=strict_discrete_set,
+        values={"On": 1, "Off": 0},
+        map_values=True
+    )
+
+    current_ac_range = Instrument.control(
+        ":CURR:AC:RANG?", ":ROUT I; :CURR:AC:RANG %g",
+        """ A property that control the current sensitivity (primary PSD), range from 10E-15, 20E-15, ... 1E-6,
+            rounding is applied to arbitrary values specified. Current sensitivity is the product of the current-voltage
+            conversion gain and the voltage sensitivity.""",
+        validator=truncated_range,
+        values=[10e-15, 1e-6]
+    )
+
+    current2_ac_range = Instrument.control(
+        ":CURR2:AC:RANG?", ":ROUT I; :CURR2:AC:RANG %g",
+        """ A property that control the current sensitivity (secondary PSD), range from 10E-15, 20E-15, ... 1E-6,
+            rounding is applied to arbitrary values specified. Current sensitivity is the product of the current-voltage
+            conversion gain and the voltage sensitivity.""",
+        validator=truncated_range,
+        values=[10e-15, 1e-6]
+    )
+
+    data = Instrument.control(
+        ":DATA?", ":DATA %d",
+        """ A property that allow user to set measurement data sets. The data sets will be able to read by the :FETC? query.
```

Comment:
Maybe it is better to refer to the `fetch` property rather than the internal `:FETC?` query.